pepper-apis/src/test/java/org/broadinstitute/ddp/route/GetActivityInstanceRouteTest.java

```diff
@@ -200,7 +200,7 @@ private static void setupActivityAndInstance(Handle handle) throws Exception {
         Template contentBody = new Template(TemplateType.HTML, null, "<p>hello body</p>");
         ContentBlockDef contentDef = new ContentBlockDef(contentTitle, contentBody);
         ContentBlockDef content2 = new ContentBlockDef(null, Template.html(
-                "<p>$DDP_PARTICIPANT_FIRST_NAME<br/>$DDP_PARTICIPANT_LAST_NAME<br/>$DDP_DASHED_DATE</p>"));
+                "<p>$ddp.participantFirstName()<br/>$ddp.participantLastName()<br/>$ddp.date(\"MM-dd-uuuu\")</p>"));
```

Comment:
We can now control how we want the date to be formatted without adding new code:
```
$ddp.date("uuuu-MM-dd")    => 2020-06-15
$ddp.date("MM-dd-uuuu")    => 06-15-2020
$ddp.date("MM/dd/uuuu")    => 06/15/2020
$ddp.date("MM/uuuu")       => 06/2020
```